titles.plugin.zsh

```diff
@@ -28,7 +28,7 @@ function _zsh_title__preexec() {
     fg)	cmd="${(z)jobtexts[${(Q)cmd[2]:-%+}]}" ;;
     %*)	cmd="${(z)jobtexts[${(Q)cmd[1]:-%+}]}" ;;
   esac
-  update_title "$cmd" "%20<...<%~"
+  update_title "$cmd" "%20<...<%~">/dev/null
```

Comment:
You're right. I think I found the culprit now. Remove prompt expansion `-P` in `print -Pn` from the inner variable and it seems to work.